internal/app/secretless/listeners/http/listener.go

```diff
@@ -7,24 +7,18 @@ import (
 	"io"
 	"io/ioutil"
 	"log"
-	"net"
 	"net/http"
 	"reflect"
 	"strconv"
 
 	"github.com/conjurinc/secretless-broker/pkg/secretless/config"
 	plugin_v1 "github.com/conjurinc/secretless-broker/pkg/secretless/plugin/v1"
-	validation "github.com/go-ozzo/ozzo-validation"
+	"github.com/go-ozzo/ozzo-validation"
```

Comment:
Third-party imports should be between local and built-in ones (even though it wasn't you that added it originally)